tests/qa_automation/qa_run.pm

```diff
@@ -80,16 +70,17 @@ sub run() {
 
     # When finished, the screen will terminate
     for (1 .. 60) {
-        my $ret = check_screen [qw/qa_screen_done qa_error/], 120;
-        if ($ret && match_has_tag('qa_error')) {
-            die "run failed";
-        }
-        elsif ($ret) {
+        if (check_screen [qw/qa_screen_done qa_error/], 120) {
+            if (match_has_tag('qa_error')) {
+                die "run failed";
+            }
             last;
         }
-        # change the screen periodically to avoid standstill detection for long running tests
-        my $time = time;
-        type_string "$time\n";
+        else {
+            # change the screen periodically to avoid standstill detection for long running tests
```

Comment:
we don't, but we still have screensavers, so it still makes somewhat sense for the video :)